sdk/cosmos/azure-cosmos-spark_3-1_2-12/README.md

```diff
@@ -123,7 +123,7 @@ or contact [opencode@microsoft.com][coc_contact] with any additional questions o
 [coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
 [coc_contact]: mailto:opencode@microsoft.com
 [azure_subscription]: https://azure.microsoft.com/free/
-[samples]: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/cosmos/azure-spring-data-cosmos/src/samples/java/com/azure/spring/data/cosmos
+[samples]: https://github.com/Azure/azure-sdk-for-java/tree/feature/spring-cloud-azure-6.x/sdk/spring/azure-spring-data-cosmos/src/samples/java/com/azure/spring/data/cosmos
```

Comment:
Shouldn't we stick with the Main branch here? My assumption was the feature branch is used for future versions - while Main contains the current stable version?
https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring/azure-spring-data-cosmos/